pages/about-coq.html

```diff
@@ -78,9 +78,9 @@
     synthetic <a href="/who-did-what-in-coq">who did what</a>
     table).</p>
 
-    <p>The development is coordinated by the ADT Coq (Action for Technological Development), that gathers the teams involved in the implementation of the Coq Proof Assistant. The teams registered in the ADT are the <a href="http://www.inria.fr">INRIA</a> projects <a href="http://www.pps.univ-paris-diderot.fr/pi.r2/">πr²</a> and <a href="http://www-sop.inria.fr/marelle/">Marelle</a>, and the team <a href="http://cedric.cnam.fr/AfficheEquipe.php?id=7&amp;lang=fr">CPR</a> from <a href="http://www.cnam.fr">CNAM</a>.
-</p>
-
+    <p>The development is currently coordinated by a core development
+    team made of Inria researchers and engineers in various
+    project-teams in France.</p>
 
     <p> Coq is written in the <a href="https://ocaml.org/index.fr.html">OCaml</a> language, with a bit of C. It is
```

Comment:
```suggestion
    <p> Coq is written in the <a href="https://ocaml.org/index.fr.html">OCaml</a> language, with a small amount of C. It is
```

Reply:
I'm not even sure this detail is interesting, and for sure it is not a distinguished feature (on the contrary OCaml has his place here).

Reply:
So just remove the mention of C?

Reply:
Removing "C" is fine by me.